docs/_providers/akamaiedgedns.md

```diff
@@ -0,0 +1,68 @@
+---
+name: AkamaiEdgeDns
+title: Akamai Edge DNS Provider
+layout: default
+jsId: AKAMAIEDGEDNS
+---
+# Akamai Edge DNS Provider
+"Akamai Edge DNS Provider" configures Akamai's
+[Edge DNS](https://www.akamai.com/us/en/products/security/edge-dns.jsp) service.
+
+This provider interacts with Edge DNS via the
+[Edge DNS Zone Management API](https://developer.akamai.com/api/cloud_security/edge_dns_zone_management/v2.html).
+
+Before you can use this provider, you need to create an "API Client" with authorization to use the
+[Edge DNS Zone Management API](https://developer.akamai.com/api/cloud_security/edge_dns_zone_management/v2.html).
+
+See the "Get Started" section of [Edge DNS Zone Management API](https://developer.akamai.com/api/cloud_security/edge_dns_zone_management/v2.html),
+which says, "To enable this API, choose the API service named DNS—Zone Record Management, and set the access level to READ-WRITE."
+
+Follow directions at [Authenticate With EdgeGrid](https://developer.akamai.com/getting-started/edgegrid) to generate
+the required credentials.
+
+## Configuration
+In the credentials file (creds.json), you must provide the following:
+{% highlight json %}
+"akamaiedgedns": {
+    "client_secret": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+    "host": "akaa-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.xxxx.akamaiapis.net",
+    "access_token": "akaa-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+    "client_token": "akaa-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+    "contract_id": "X-XXXX",
+    "group_id": "NNNNNN"
+}
+{% endhighlight %}
+
+## Usage
+A new zone created by DNSControl:
+```
```

Comment:
add a blank line before this and others like it.

Reply:
Done